src/content/developers/docs/smart-contracts/languages/index.md

```diff
@@ -198,6 +200,49 @@ def endAuction():
 
 This example should give you a sense of what Vyper contract syntax is like. For a more detailed description of the functions and variables, [see the docs](https://vyper.readthedocs.io/en/latest/vyper-by-example.html#simple-open-auction).
 
+## Yul and Yul+ {#yul}
+
+**Yul**
+- Intermediate language for Ethereum.
+- Support for EVM 1.0, EVM 1.5 and eWASM is planned, and it is designed to be a usable common denominator of all three platforms.
+- Good target for high-level optimisation stages that can benefit all target platforms equally.
+
+**Yul+**
+- A low-level, highly efficient extension to Yul.
+- Initially designed for an optimistic rollup contract.
+- Yul+ can be looked at as an experimental upgrade proposal to Yul, adding new features to it.
+
+### Important links {#important-links-2}
+- [Yul Documentation](https://solidity.readthedocs.io/en/latest/yul.html)
+- [Yul+ Documentation](https://github.com/fuellabs/yulp)
+- [Yul+ Playground](https://yulp.fuel.sh/)
+- [Yul+ Introduction Post](https://medium.com/@fuellabs/introducing-yul-a-new-low-level-language-for-ethereum-aa64ce89512f)
+
+### Example contract {#example-contract-2}
+
+The following simple example can be compiled using ``solc --strict-assembly``. 
```

Comment:
Maybe add `--bin` because otherwise it will not output anything. Or provide a full commandline:
``solc --strict-assembly --bin input.yul``
and meniton taht the example should be stored in the file `input.yul`